dataprofiler/tests/data/txt/utf8.txt

```diff
@@ -1,4 +1,4 @@
-première is first
+première is first. ....
 première is slightly different
-Кириллица is Cyrillic
-𐐀 am Deseret
+Кириллица is Cyrillic. . . test
+𐐀 am Deseret. . test
```

Comment:
Do we test a blank file or file with just some enters or something?

Reply:
adding `empty.txt` to the file check list for our tests